nethsm/__init__.py

```diff
@@ -387,7 +390,10 @@ def request(
 
         body: Union[str, bytes, None] = None
         if data:
-            body = data.read(-1)
+            if isinstance(data, bytes):
+                body = data
+            else:
+                body = data.read(-1)
```

Comment:
The `-1` can be ommited, as read by default reads until EOF is reached.
I.e. the following could should suffice.
```
body = data.read()
```

Reply:
good point, fixed.